R/descriptives.R

```diff
@@ -1228,9 +1228,21 @@ Descriptives <- function(jaspResults, dataset, options) {
 
 .descriptivesDotPlots_SubFunc <- function(dataset, variable, title){
   
-  dotplot <- createJaspPlot(title = title)
-  
+  dotPlot <- createJaspPlot(title = title)
   x <- na.omit(dataset[[variable]])
+  x <- x[is.finite(x)]
+  
+  if (length(x) == 0) {
+    dotPlot$setError(gettext("No non-missing values!"))
+    return(dotPlot)
+  }
+  
+  if (length(unique(x)) == 1) {
+    dotsize <- .03
+  } else{
+    dotsize <- 1
+  }
```

Comment:
so for one observation, I get this:

![image](https://user-images.githubusercontent.com/21319932/108388896-66d14a00-720f-11eb-8e40-c67efcea7551.png)

maybe this adjustment only needs to happen when it's not a factor?